### PR TITLE
Bring documentation look and feel in line with bokehplots.com

### LINKED ISSUE #3388: Bring documentation styling in line with bokehplots.com
The new bokehplots.com has been designed by @SaraSchnadt, the documentation should:
* share the header and footer to make it seem like a seamless experience
* have a new top nav to reflect the tightened up doc structure (#3367)
* share fonts, and look and feel, with bokehplots.com

### PR DESCRIPTION
**Pull request by birdsarah**
Fixes #3388

### CODE REVIEW

**Comment by birdsarah** [comment]
This is ready @bokeh/core, would really appreciate people downloading and trying to build to make sure this works. Also, the real test will be in dpeloyment, I think this should all deploy fine, but @damianavila I'd like to propose a dev build once this gets merged so we can check it all works live.

**Comment by damianavila** [comment]
>but @damianavila I'd like to propose a dev build once this gets merged so we can check it all works live.

Just ping me on flowdock and I will trigger the dev build...

**Comment by birdsarah** [comment]
@bryevdv @damianavila - can you clear the cache and restart the build - it shouldn't be looking for that template any more.

**Comment by bryevdv** [comment]
done

**Comment by birdsarah** [comment]
@bryevdv @damianavila well that didn't seem to help, any thoughts?

File: sphinx/source/conf.py [hunk 1]
```diff
@@ -116,104 +118,50 @@
...
-# If not '', a 'Last updated on:' timestamp is inserted at every page bottom,
-# using the given strftime format.
-#html_last_updated_fmt = '%b %d, %Y'
-
-# If true, SmartyPants will be used to convert quotes and dashes to
-# typographically correct entities.
-#html_use_smartypants = True
 
 # Custom sidebar templates, maps document names to template names.
 standard_sidebars = ['sidebartoc.html', 'sourcelink.html', 'searchbox.html']
```
Comment (bryevdv) [inline sphinx/source/conf.py]: @birdsarah here?

Comment (damianavila) [inline sphinx/source/conf.py]: that's a good guess...

**Comment by birdsarah** [comment]
Thanks @bryevdv - that was it!

**Comment by birdsarah** [comment]
Note adding flask to meta.yaml has nothing to do with this PR, but it somehow was missing from master, so needed to make the tests on this PR run.

**Pull request merged**